File: randomColor.lua
local function random_number(min_val, max_val)
    if min_val > max_val then
        min_val, max_val = max_val, min_val
    end
    return math.random(min_val, max_val)
end

local function random_color()
    local r = random_number(0, 255)
    local g = random_number(0, 255)
    local b = random_number(0, 255)
    return string.format("rgb(%d, %d, %d)", r, g, b)
end

local color = random_color()
print("Random color:", color)

File: randomColor.py
import random

def random_number(min_val, max_val):
    if min_val > max_val:
        min_val, max_val = max_val, min_val
    return random.randint(min_val, max_val)

def random_color():
    r = random_number(0, 255)
    g = random_number(0, 255)
    b = random_number(0, 255)
    return f"rgb({r}, {g}, {b})"

color = random_color()
print("Random color:", color)
